config/sample_application.yml

```diff
@@ -1,5 +1,5 @@
-GITHUB_KEY:
-GITHUB_SECRET:
+GITHUB_KEY: 'c8baaa499b1b6819538c'
+GITHUB_SECRET: '3fdbe8a325b60f17a3fbbba56eb9390435a45c4c'
```

Comment:
This two secret should probably be deleted from the history

Reply:
Thanks for catching this @rhymes. This was in the initial PR, but I had Sophia remove the file. The secrets are no longer valid, but definitely that file should be reverted. I think this file creeped backed in by accident with the most recent changes. My bad for not noticing that in the most recent changes.

Reply:
I've deleted the keys from the file and have deactivated those keys. Let me know if I should undo the commit with the keys. Thanks!